src/java.base/share/classes/java/net/SocketOptions.java

```diff
@@ -29,15 +29,15 @@
 
 /**
  * Interface of methods to get/set socket options.  This interface is
- * implemented by: <B>SocketImpl</B> and  <B>DatagramSocketImpl</B>.
- * Subclasses of these should override the methods
- * of this interface in order to support their own options.
+ * implemented by {@link SocketImpl} and {@link DatagramSocketImpl}.
+ * Subclasses of these two classes should override the {@link #getOption(int)} and
+ * {@link #setOption(int, Object)} methods of this interface in order to support their own options.
  * <P>
- * The methods and constants which specify options in this interface are
- * for implementation only.  If you're not subclassing SocketImpl or
- * DatagramSocketImpl, <B>you won't use these directly.</B> There are
- * type-safe methods to get/set each of these options in Socket, ServerSocket,
- * DatagramSocket and MulticastSocket.
+ * The methods and constants defined in this interface are
+ * for internal implementation use only. If you're not subclassing {@code SocketImpl} or
```

Comment:
I don't think we can insert the word "internal" here it's part of the exposed interface for this legacy mechanism.

Reply:
I've now updated the PR to remove the "internal" word that was introduced as part of this change.